control/trajectory_follower_nodes/param/lateral_controller_defaults.param.yaml

```diff
@@ -56,7 +56,9 @@
     stop_state_entry_ego_speed: 0.001
     stop_state_entry_target_speed: 0.001
     converged_steer_rad: 0.1
-    check_steer_converged_for_stopped_state: true
+    keep_steer_control_until_converged: true
```

Comment:
better to set false by default

Reply:
[b51a021](https://github.com/autowarefoundation/autoware.universe/pull/901/commits/b51a021429fcd6bd41642c52866f154b4797bdc6)